src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_routes.erl

```diff
@@ -280,19 +280,5 @@ make_args(Item, _, OrgName) ->
     [OrgName, ej:get({<<"name">>}, Item)].
 
 %% extract name
-
-%% When data_bag_items are stored in couchdb, they are wrapped in cruft such that the actual
-%% item is under the 'raw_data' key of the wrapper. When data_bag_items are stored in SQL,
-%% just the item is stored. This helper function extracts the id of the data_bag_item for
-%% either type. We could have used darklaunch here, but since there are two cases and it is
-%% easy to test for, we just inspect the data at hand. This can go away once data_bag_items
-%% are only in SQL.
-%% TODO: when data bags are no longer in couchdb, clean this
 data_bag_item_id(Item) ->
-    case ej:get({<<"id">>}, Item) of
-        undefined ->
-            %% we must have a structure coming out of couchdb wrapped in data_bag_item cruft
```

Comment:
I had left this one in on my branch because I figured it was possible we still had old data out there that fit it - but looking closer , I think removing it is correct, since this would have been changed/fixed during the original migration.